ReleaseNotes-2.9

```diff
@@ -58,6 +58,12 @@ Wanted changes:
 
  * The editing of a user's realname, email adress or password is no longer possible if LDAP mode is activated
 
+ * Unused ldap options in options.yml were dropped:
```

Comment:
They are *unused* because of the point above right? Please connect those two entries.